data-es7.js

```diff
@@ -773,7 +801,9 @@ exports.tests = [
   category: 'strawman',
   link: 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Set([1, 2, 3, 2, 1])) === '[1,2,3]';
+    var set = new Set();
+    [1, 2, 3, 2, 1].forEach(function (i) { set.add(i); });
+    return JSON.stringify(set) === '[1,2,3]';
```

Comment:
What I'd like to see in a `stringify` test is whether the contents are *recursively* serialised - for instance, whether an object inside a Set is serialised as well. But, it's not really that important.